docs/custom_conf.py

```diff
@@ -334,3 +330,85 @@ def _compute_navigation_tree(context):
     ('py:class', 'ops.testing.CharmType'),
     ('py:obj', 'ops.testing.CharmType'),
 ]
+
+
+# This is very strongly based on
```

Comment:
I guess this approach is recommended at https://github.com/picnixz/sphinx/blob/8c6d234e961eaf945567755863d377fa27dc6f22/doc/development/tutorials/examples/todo.py#L41